spring-boot/admin/src/main/java/eu/coatrack/admin/service/OAuthUserDetailsService.java

```diff
@@ -56,19 +73,29 @@ private OAuth2User getLoggedInUser() {
     }
 
     public String getLoginNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("login");
+        return getLoggedInUser().getAttribute(attributeNameForLogin);
     }
 
     public String getNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("name");
+        return getLoggedInUser().getAttribute(attributeNameForName);
```

Comment:
```suggestion
        return getLoggedInUser().getAttribute(attributeNameForUserName);
```

Reply:
This was already solved in the threads before, based on the suggestions from Gunnar